R/module-template.R

```diff
@@ -218,19 +218,22 @@ defineModule(sim, list(
     defineParameter(\".saveInitialTime\", \"numeric\", NA, NA, NA, \"This describes the simulation time at which the first save event should occur\"),
     defineParameter(\".saveInterval\", \"numeric\", NA, NA, NA, \"This describes the simulation time interval between save events\")
   ),
-  inputObjects = data.frame(
-    objectName = NA_character_,
-    objectClass = NA_character_,
-    sourceURL = \"\",
-    other = NA_character_,
-    stringsAsFactors = FALSE
-  ),
-  outputObjects = data.frame(
-    objectName = NA_character_,
-    objectClass = NA_character_,
-    other = NA_character_,
-    stringsAsFactors = FALSE
-  )","
+  inputObjects = data.frame(rbindlist(list(
+    #expectsInput(objectName, objectClass, desc, sourceURL, ...), # desc is description; default is NA for sourceURL
+                                                                  # other information can be added through ... arguement
+    expectsInput(objectName = NA_character_, objectClass = NA_character_, desc = NA_character_,
+                sourceURL = NA_character_, otherInformation = NA_character_) # should be replaced or deleted
+    #expectsInput(objectName = \"testmap\", objectClass = \"RasterLayer\", desc = \"this is a test map for input object1\",
```

Comment:
there is already a commented out example above. do we really need both? I think we should pick one of them.